packages/mui-base/src/SliderUnstyled/SliderUnstyled.test.tsx

```diff
@@ -136,31 +152,43 @@ describe('<SliderUnstyled />', () => {
     });
   });
 
-  [
+  type Values = Array<[string, number[]]>;
+
+  const values = [
     ['readonly range', Object.freeze([2, 1])],
     ['range', [2, 1]],
-  ].forEach(([valueLabel, value]) => {
+  ] as Values;
+  values.forEach(([valueLabel, value]) => {
     it(`calls onChange even if the ${valueLabel} did not change`, () => {
       const handleChange = spy();
+
       const { container } = render(
         <SliderUnstyled min={0} max={5} onChange={handleChange} value={value} />,
       );
-      stub(container.firstChild, 'getBoundingClientRect').callsFake(() => ({
-        width: 100,
-        height: 10,
-        bottom: 10,
-        left: 0,
-      }));
+
+      if (container.firstChild !== null) {
+        // container.firstChild doesn't have getBoundingClientRect method. hence typescript is throwing error.
+        // @ts-expect-error
```

Comment:
You can inline this with `ts-expect-error`:
```suggestion
        // @ts-expect-error - container.firstChild doesn't have getBoundingClientRect method. 
```